src/main/java/racing/Racing.java

```diff
@@ -0,0 +1,32 @@
+package racing;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Racing {
+     public static void main(String[] args) {
+
+        CarRace carRace = new CarRace();
+        ArrayList<Car> carArrayList = new ArrayList<>();
+        List<String> carList = carRace.getRacingJoinList(); // 자동차 경주 참가자 명단 입력
```

Comment:
```suggestion
        List<Car> carArrayList = new ArrayList<>();
        List<String> carList = carRace.getRacingJoinList(); // 자동차 경주 참가자 명단 입력
```

특별한 이유가 없다면 구현체보다 인터페이스를 사용하는 것이 좋습니다. 
변수명도 고민해 보시면 좋을 것 같아요. `List<Car>` 에서 우리의 관심사는 List인가요? Car가 복수개 있다는 것 아닐까요? 그럼 cars가 조금 더 자연스러울 것 같습니다. 변수명에서 자료구조를 사용하면 자료구조가 변경되는 경우 모두 변경해야 합니다.